Documentation/gettingstarted/clustermesh/services.rst

```diff
@@ -110,3 +142,61 @@ Deploying a Simple Example Service
       kubectl exec -ti deployment/x-wing -- curl rebel-base
 
    You will see replies from pods in both clusters again.
+
+9. In cluster 1, add ``io.cilium/service-affinity="local"`` to existing global service
```

Comment:
I think this should be a separate section? This makes it look like it requires 16 steps just to get a global service running :thinking: